corehq/motech/repeaters/templates/repeaters/partials/repeater_row.html

```diff
@@ -16,9 +16,14 @@
             {{ repeater.count_State.Fail }}
         </a>
     </td>
+    <td>
+        <a href="{% url 'domain_report_dispatcher' domain report %}?repeater={{ repeater.repeater_id }}&amp;record_state=INVALIDPAYLOAD">
```

Comment:
Nit: Could this state be separated with a _ as `INVALID_PAYLOAD`? It makes it a little easier to read, but fine to keep as is if this is how we define the other states.

Reply:
That is the value of the "Record State" select box. It is [generated here](https://github.com/dimagi/commcare-hq/blob/a0639babb843b3063d5c563263818d8c7dfe8d5a/corehq/apps/reports/filters/select.py#L142) using `s.name.upper()` where `s` is a State.

We could write a fun little camelcase-to-snakecase converter ...
```python
def camel_to_snake(value):
    # Insert an underscore between a lowercase letter and an uppercase
    # letter. Then convert to lowercase.
    return re.sub(r'(?<!^)(?=[A-Z])', '_', value).lower()
```
... and then call that instead of `upper()` to set the select box values ... but that feels like a lot to make one of the five possible values of a request param more readable. I feel like that missing underscore outweighs the addition of an extra function (even though that is a sweet regular expression :D (thanks ChatGPT)).

Reply:
Ah I see, so it's just camelcase that was taken through `upper()`. In that case, especially given the LOE to turn it into snake case, I think it's fine to leave as is. Thanks for the thorough explanation! (Also +1 for ChatGPT)

Reply:
It turns out that `RECORD_INVALIDPAYLOAD_STATE` is also a thing. :face_with_diagonal_mouth: I think I'm going to leave it as is.